axelor-base/src/main/java/com/axelor/apps/base/service/LanguageServiceImpl.java

```diff
@@ -0,0 +1,197 @@
+                    + "SET order_seq = order_seq - 1 "
+                    + "FROM meta_select "
+                    + "WHERE meta_select.id = meta_select_item.select_id "
+                    + "AND meta_select_item.order_seq > :orderNumber "
+                    + "AND meta_select.name = :selectName "
+                    + "AND meta_select.module = :moduleName ")
+            .setParameter("orderNumber", orderNumber)
+            .setParameter("selectName", SELECT_NAME)
+            .setParameter("moduleName", NEW_SELECT_LANGUAGE_MODULE);
+    query.executeUpdate();
+  }
+
+  /**
+   * Fetch the native MetaSelect and the overriddenMetaSelect. Copy MetaSelectItems from the native
+   * to the new one. If it is the first time we add a new Language object, we should create a new
+   * MetaSelect object with a name field of "select.language", a module field of "axelor-base" and a
+   * priority field of the original priority + 1. Then, we should copy all the existing
+   * MetaSelectItems from the original MetaSelect to the new MetaSelect.
+   */
+  @Transactional
+  @Override
+  public MetaSelect copyMetaSelect() {
+    // fetch the native MetaSelect
+    Query<MetaSelect> metaSelectQuery =
+        metaSelectRepository
+            .all()
+            .filter("self.name = :selectName and self.module = :nativeModuleName");
+    metaSelectQuery
+        .bind("selectName", SELECT_NAME)
+        .bind("nativeModuleName", NATIVE_SELECT_LANGUAGE_MODULE);
+    MetaSelect nativeMetaSelect = metaSelectQuery.fetchOne();
+    if (nativeMetaSelect == null) return null;
+    MetaSelect overriddenMetaSelect = getOverriddenMetaSelectFromNativeMetaSelect(nativeMetaSelect);
+    System.out.println("overriddenMetaSelect:" + overriddenMetaSelect);
+    metaSelectRepository.save(overriddenMetaSelect);
+    return overriddenMetaSelect;
+  }
+
+  /**
+   * create a new MetaSelect obj that has the same metaSelectItems as the native one.
+   *
+   * @param nativeMetaSelect
+   * @return overriddenMetaSelect
+   */
+  private MetaSelect getOverriddenMetaSelectFromNativeMetaSelect(MetaSelect nativeMetaSelect) {
+    MetaSelect overriddenMetaSelect = new MetaSelect();
+    Integer oldPriority = nativeMetaSelect.getPriority();
+    overriddenMetaSelect.setName(SELECT_NAME);
+    overriddenMetaSelect.setModule(NEW_SELECT_LANGUAGE_MODULE);
+    overriddenMetaSelect.setPriority(oldPriority + 1);
+
+    List<MetaSelectItem> nativeMetaSelectItemList = nativeMetaSelect.getItems();
+    MetaSelectItem newMetaSelectItem;
+    for (MetaSelectItem nativeMetaSelectItem : nativeMetaSelectItemList) {
+      newMetaSelectItem = new MetaSelectItem();
+      newMetaSelectItem.setTitle(nativeMetaSelectItem.getTitle());
+      newMetaSelectItem.setValue(nativeMetaSelectItem.getValue());
+      newMetaSelectItem.setOrder(nativeMetaSelectItem.getOrder());
+      overriddenMetaSelect.addItem(newMetaSelectItem);
+      System.out.println(overriddenMetaSelect);
```

Comment:
Don't use System.out.println, if you want to put some logs, use the Axelor Logger